packages/jest-util/src/index.js

```diff
@@ -74,6 +76,13 @@ const getPackageRoot = () => {
   return root;
 };
 
+const getModulePath = (from: Path, moduleName: string) => {
+  if (moduleName[0] !== '.' || path.isAbsolute(moduleName)) {
+    return moduleName;
+  }
+  return path.normalize(path.dirname(from) + '/' + moduleName);
+};
+
```

Comment:
Instead of duplicating it here I'd really prefer to add `Resolver.getModulePath` and re-using the static method both in `jest-resolve` and `jest-runtime`.